crates/biome_js_analyze/src/lint/nursery/use_jsx_img.rs

```diff
@@ -0,0 +1,94 @@
+use biome_analyze::{
+    Ast, FixKind, Rule, RuleDiagnostic, RuleDomain, RuleSource, declare_lint_rule,
+};
+use biome_console::markup;
+use biome_diagnostics::Severity;
+use biome_js_syntax::jsx_ext::AnyJsxElement;
+use biome_rowan::AstNode;
+use biome_rule_options::use_jsx_img::UseJsxImgOptions;
+
+declare_lint_rule! {
+    /// For performance reasons, always provide width and height attributes for `<img>` elements; it will help to prevent layout shifts.
```

Comment:
As explained in the [contribution guide](https://github.com/biomejs/biome/blob/main/crates/biome_analyze/CONTRIBUTING.md#general-structure), the first paragraph must be used as a brief explanation of the rule. This paragraph doesn't seem to be a brief explanation. Can you update it?